tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "target": "es6"
+  },
+  "include": [
+    "./src/index.ts",
```

This could be all TS files in the project:

```suggestion
    "./**/*.ts"
```

---

I did `src/*.ts` in 02d2728